support/src/test/java/com/shopify/graphql/support/IntegrationTest.java

```diff
@@ -168,7 +169,7 @@ public void testOptionalFieldOnInput() throws Exception {
     @Test
     public void testUnsetOptionalFieldOnInput() throws Exception {
```

Comment:
This test should probably be renamed to `testOptionalFieldOnInputAsUndefined`. It would also be worth having tests `testOptionalFieldOnInputAsExplicitNull` and `testOptionalFieldOnInputAsInputValue.`

Reply:
Okie. I will add them.